### PR TITLE
Added Husky&Prettier to the contracts dir to mirror dApp - :beer:

### LINKED ISSUE #533: Prettier enforcement on JS in contracts subdir
I think we have this via a Husky hook on the DApp stuff. Would be great to add it to the contracts dir as well as unpretty js keeps slipping through.

### PR DESCRIPTION
**Pull request by cipherzzz**
**Issue**:
resolves #533 

**Description**: 
I mirrored the husky/prettier hook from the `dApp` project to the `contract` project.

**Changes**:
- Added Husky:^4.3.0 to devDependencies
- Added husky hook for `pre-push`
- added all the missing js directories in alphabetical order to the `prettier` package.json scripts
- pushed with `no-verify` to keep footprint small(bypass husky)

**Testing**:
- merge pr into your branch and attempt to push branch to origin. The husky hook should run.

### CODE REVIEW

**Comment by cipherzzz** [comment]
I don't know why this is failing:
```
1) Compensation Claims
       User claims
         "before each" hook for "should show a user their funds, and a total of all funds":
     Error: Timeout of 20000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves. (/home/runner/work/origin-dollar/origin-dollar/contracts/test/compensation/compensationClaims.js) 
```
I didn't touch that code and I didn't format any js.

**Comment by franckc** [comment]
The `Compensation Claims` is flaky. Not related to the changes in this PR, your can ignore.

File: contracts/package.json [hunk 1]
```diff
@@ -30,8 +31,8 @@
     "@nomiclabs/hardhat-ethers": "^2.0.0",
     "@nomiclabs/hardhat-solhint": "^2.0.0",
     "@nomiclabs/hardhat-waffle": "^2.0.0",
-    "@openzeppelin/hardhat-upgrades": "^1.6.0",
     "@openzeppelin/contracts": "2.5.0",
+    "@openzeppelin/hardhat-upgrades": "^1.6.0",
```
Comment (franckc) [inline contracts/package.json:35]: haha. I see you re-ordered so that those packages are in alphabetical order. I'm OCD too... so thank you :)

**Review by franckc — APPROVED** [review]
Nice! Thanks for adding this. LGTM

**Pull request merged**